gui/widgets/label.py

```diff
@@ -6,9 +6,13 @@
 from gui.core.nanogui import DObject
 from gui.core.writer import Writer
 
+ALIGN_LEFT = 0
```

Comment:
Please use `const()` in these declarations to save bytes.

Reply:
Added